annotations/templates/annotations/project_details.html

```diff
@@ -42,7 +42,7 @@
         <thead>
             <th class="col-xs-9">
                 <a class="h4" style="text-decoration: none; vertical-align: middle;" href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}" id="order_title">Title</a>
-                <a class="btn btn-sm btn-default" aria-label="Left Align" href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}">
+                <a class="btn btn-sm btn-default" aria-label="Left Align" href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}">\
```

Comment:
The \ symbol is after the anchor tag ends. Can you remove it or put it inside the anchor tag ?